src/js/bs3/module/HelpDialog.js

```diff
@@ -7,6 +10,31 @@ define([], function () {
     var options = context.options;
     var lang = options.langInfo;
 
+
+    this.createShortCutList = function () {
+      var keyMap = options.keyMap[agent.isMac ? 'mac' : 'pc'];
+
+      var $list = $('<div />');
+      for (var keyString in keyMap) {
```

Comment:
If you use `for in` with object, you should check hasOwnProperty.
http://stackoverflow.com/questions/684672/loop-through-javascript-object

or
```
Object.keys(keyMap).forEach(function (key) {
   // key
});
```